assets/css/utilities/_borders.scss

```diff
@@ -13,6 +13,12 @@
   }
 }
 
+.border-desktop {
+  @media (min-width: 767px) {
+    border: 1px solid $border-grey;
+  }
+}
+
```

Comment:
Does this get used or https://github.com/blockchain/My-Wallet-V3-Frontend/pull/1037/files#diff-1eb8d7b43d1798135358050888f1357dR132?

Should also be min-width: 768px